docs/tutorials/azure-quantum/getting_started_honeywell.ipynb

```diff
@@ -0,0 +1,356 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f2adf23",
+   "metadata": {},
+   "source": [
+    "## Asynchronous workflow using Jobs\n",
+    "\n",
+    "For long-running circuits, it can be useful to run them asynchronously. The `service.create_job` method returns a `Job`, which you can use to get the results after the job has run successfully."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "d112ddb3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = service.create_job(\n",
+    "    program=circuit,\n",
+    "    repetitions=100\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4ef4b6fc",
+   "metadata": {},
+   "source": [
+    "To check on the job status, use `job.status()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "2969c0d8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Waiting'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "job.status()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5695c4b",
+   "metadata": {},
+   "source": [
+    "To wait for the job to be done and get the results, use the blocking call `job.results()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "6af542d1",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      ".......{'m_b': ['00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00', '00']}\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = job.results()\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54392f7a",
+   "metadata": {},
+   "source": [
+    "Note that this does not return a `cirq.Result` object. Instead it returns a dictionary that is specific to the Honeywell simulator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "72547953",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "dict"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(result)"
```

Comment:
Rather than call type(result) and return 'dict' which is not particularly helpful, it may be worth explaining what kind of dict this is (what are the keys and values?).  Alternatively, we could link to documentation that explains it.  Same comment on the IonQ page.